caffe2/python/optimizer_test.py

```diff
@@ -21,6 +23,19 @@ def check_optimizer(self, optimizer):
             np.testing.assert_allclose(np.array([1.0]), tensor, atol=1e-5)
 
 
+class TestMultiPrecisionSgd(OptimizerTestBase, TestCase):
+    def build_optimizer(self, model):
+        self._skip_gpu = False
+        return build_multi_precision_sgd(model, base_learning_rate=0.1)
```

Comment:
will this actually exercise the logic? I think if you don't use an fp16 initializer , then you will just fallback to the normal sgd...